Classes/EventStore/Normalizer/ValueObjectNormalizer.php

```diff
@@ -21,6 +21,9 @@
  */
 final class ValueObjectNormalizer implements DenormalizerInterface
 {
+    protected $resolveConstructorMethodCache = [];
+    protected $resolveNamedConstructorMethodCache = [];
```

Comment:
```suggestion
    /**
     * @var array
     */
    private $resolveNamedConstructorMethodCache = [];
```

Reply:
Done

Reply:
Sorry didn't notice that you want the property private - finally done :)